### PR TITLE
qa_crowbarsetup: use crowbarctl to commit proposals

### PR DESCRIPTION
**Pull request by jdsn**
Switch to crowbarctl for crowbar proposal commit from cloud6 on.

### CODE REVIEW

**Comment by jdsn** [comment]
Background: the commit times out often, therefore use crowbarctl which has different timeout.

All other calls to `crowbar ...` should be done as well in a later step. This PR is needed to get the gating of the SAP pull requests working.

File: scripts/qa_crowbarsetup.sh [hunk 1]
```diff
@@ -3170,6 +3170,18 @@ function set_noproxyvar
...
 
+# commit a proposal, but use crowbarctl from cloud6 on
+function crowbar_proposal_commit
+{
+    local proposal="$1"
+    local proposaltype="${2:-default}"
+    if iscloudver 6plus ; then
+        safely crowbarctl proposal commit "$proposal" "$proposaltype"
+    else
+        safely crowbar "$proposal" proposal commit "$proposaltype"
```
Comment (bmwiedemann) [inline scripts/qa_crowbarsetup.sh]: safely here will make the pre-existing "complain" useless

Comment (jdsn) [inline scripts/qa_crowbarsetup.sh]: thanks forgot to drop it, fixed now

**Comment by vuntz** [comment]
crowbarctl has a 5 minutes timeout, it's not really better... See https://ci.suse.de/job/openstack-mkcloud/59102/console and https://bugzilla.suse.com/show_bug.cgi?id=997293 and https://bugzilla.suse.com/show_bug.cgi?id=988030

**Comment by vuntz** [comment]
I submitted a patch to revert this as this kills our gating and CI in general: https://github.com/SUSE-Cloud/automation/pull/1857

**Pull request merged**